### PR TITLE
Update Workflows

### PR DESCRIPTION
**Pull request by zkamvar**
update workflows for universe release (to be merged after release)

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -16,19 +16,31 @@ knitr::opts_chunk$set(
...
 <!-- badges: end -->
 
 The goal of hubCI is to provide functionality for setting up hubverse Continuous Integration workflows.
 
 ## Installation
 
-You can install the development version of hubCI like so:
+### Latest
+
+You can install the [latest version of hubCI from the R-universe](https://hubverse-org.r-universe.dev/hubAdmin):
```
Comment (elray1) [inline README.Rmd:33]: Should this link be pointing to hubCI?

**Review by elray1 — APPROVED** [review]
approve subject to readme link update

File: README.Rmd (hunk 1 above)
Comment (zkamvar) [reply README.Rmd:33]:
```suggestion
You can install the [latest version of hubCI from the R-universe](https://hubverse-org.r-universe.dev/hubCI):
```

File: README.md [hunk 2]
```diff
@@ -12,14 +12,28 @@ status](https://www.r-pkg.org/badges/version/hubCI)](https://CRAN.R-project.org/
...
 The goal of hubCI is to provide functionality for setting up hubverse
 Continuous Integration workflows.
 
 ## Installation
 
-You can install the development version of hubCI like so:
+### Latest
+
+You can install the [latest version of hubCI from the
+R-universe](https://hubverse-org.r-universe.dev/hubAdmin):
```
Comment (zkamvar) [inline README.md:26]:
```suggestion
R-universe](https://hubverse-org.r-universe.dev/hubCI):
```

**Pull request merged**